src/mcts.py

```diff
@@ -17,45 +17,34 @@ def getBestMove(self):
         """
         current_playouts = 0
         
-        while current_playouts < self.max_playouts:
+        while current_playouts <= self.max_playouts:
```

Comment:
If we're being picky here, it should actually be `<`, since `current_playouts` starts at 0.

Reply:
I did this for usability. Since I changed the iteration on which nodes were visited, now a single playout trivially expands the root node, and none of the children. The <= just does 1 more expansion to get through that so the behavior is maintained.

Reply:
If that's the case, we should change `current_playouts` to 1.  One of the tests done at the end of each epoch is a test against an mcts with a depth of 1, and this change would impact the legitimacy of that test.

Reply:
This actually maintains the legitimacy of that test, ye?

If we keep <, then we expand and eval the root node and quit. We don't visit any children.
With <=, we expand the root, get the priors, then pick a child, ye?

Reply:
Fair enough, I'll accept this commit.